core/shared/src/main/scala/com/softwaremill/sttp/SttpApi.scala

```diff
@@ -0,0 +1,233 @@
+package com.softwaremill.sttp
+
+import java.io.{ByteArrayOutputStream, InputStream, OutputStream }
+import java.nio.ByteBuffer
+
+import com.softwaremill.sttp.internal.SttpFile
+
+import scala.annotation.{implicitNotFound, tailrec}
+import scala.collection.immutable.Seq
+import scala.concurrent.duration._
+import scala.language.higherKinds
+
+trait SttpApi extends sttpExtensions {
```

Comment:
Maybe `sttpExtensions` could now become `SttpApiExtensions`?

Reply:
Maybe, but this was the existing name so I feel changing it is kind of orthogonal to this PR.

Reply:
That was for consistency, as previously these were extensions to the `sttp` package object. Hence the lower-case name prefix :)